wiki/markdown/query.md

```diff
@@ -164,7 +159,7 @@ Example:
 
     {
       "criteria": {
-         "object.identity":"pkg:maven/com.mycompany.myproduct/sub-system@1.1.0"
+         "identity":"pkg:maven/com.mycompany.myproduct/sub-system@1.1.0"
       },
       "filter" : "incomplete_path_filter(@, 'svnIdentifier')"
```

Comment:
Since we don't have the svnIdentifier left we should use gitIdentifier here.

Reply:
done